setup.cfg

```
@@ -74,6 +74,7 @@ install_requires =
     pyzmq==22.*
     # https://github.com/pypa/setuptools/issues/3802
     setuptools>=49, <67
+    importlib_metadata
```

Looks like this library was added in Py 3.8, can we omit this dependency for 3.8 and above?

```suggestion
    importlib_metadata; python_version < "3.8"
```